src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -337,6 +349,18 @@ public boolean registerEJBReceiver(final EJBReceiver receiver) {
         return this.registerEJBReceiver(receiver, null);
     }
 
+    public boolean registerEJBReceiver(final EJBReceiver receiver, boolean deleteExistingReceiver) {
```

Comment:
Overloads do count.  In the interests of time though, could you just mark it @Deprecated?